### DISCUSSION TITLE
Change Field Name / Type after creation

### DISCUSSION THREAD

**Discussion by pateljoel** [directus/directus#14037] [Archive]
I have recently went into production with directus which is fantastic. However when I try and edit a field since I realised I made a mistake to schema, I am unable to.

Would it be possible to allow directus to edit fields after creation?

**Comment by benhaynes** [comment]
I believe there's already a discussion for this somewhere, it's definitely something we want to support, but isn't in yet.

**Comment by rowild** [comment]
@pateljoel In the meantime you can rename a field using the "Translations" option (which currently is the recommended way-to-go as far as I know).

**Reply by Sannagith** [reply]
Awesome trick, great.
Many thanks, rowild

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/2711
